src/site/content/en/angular/accessible-angular-with-codelyzer/index.md

```diff
@@ -0,0 +1,78 @@
+
+```json/6-15
+{
+  "rulesDirectory": [
+    "codelyzer"
+  ],
+  "rules": {
+    ...,
+    "template-accessibility-alt-text": true,
+    "template-accessibility-elements-content": true,
+    "template-accessibility-label-for": true,
+    "template-accessibility-tabindex-no-positive": true,
+    "template-accessibility-table-scope": true,
+    "template-accessibility-valid-aria": true,
+    "template-click-events-have-key-events": true,
+    "template-mouse-events-have-key-events": true,
+    "template-no-autofocus": true,
+    "template-no-distracting-elements": true
+  }
+}
+```
+
+TSLint works with all popular text editors and IDEs. To use it with VSCode, install the [tslint plugin](https://marketplace.visualstudio.com/items?itemName=eg2.tslint). In WebStorm TSLint is enabled by default. For other editors, check the tslint [README](https://github.com/palantir/tslint#tslint).
+
+With codelyzer’s accessibility checks set up, you get a popup showing accessibility errors in TypeScript files or inline templates as you code.
+
+![Linting in editor](editor-template.png "Linting in the editor")
+
+A codelyzer popup showing a form element labeling error.
+
+To perform linting over the entire project (including external templates), use the `ng lint` command:
+
+![Linting with Angular CLI](ng-lint.png "Linting with Angular CLI")
+
+## Supplementing codelyzer
+
+[Lighthouse](https://developers.google.com/web/tools/lighthouse/) is another tool you can use to enforce accessibility practices in your Angular application. The main difference between codelyzer and Lighthouse is when their checks get performed. Codelyzer statically analyzes the application at development time, without running it. This means that during development you can get direct feedback in your text editor or in the terminal. By contrast, Lighthouse actually runs your application and performs a bunch of checks using dynamic analysis.
+
+Both tools can be useful parts of your development flow. Lighthouse has better coverage given the checks it performs, while codelyzer allows you to iterate faster by getting constant feedback in your text editor.
+
+## Enforcing accessibility checks in your continuous integration
+
+Introducing static accessibility checks in your continuous integration (CI) can be a great enhancement for your development flow. With codelyzer you can easily enforce certain accessibility rules or other practices by running `ng lint` on each code modification (for example for each new pull request).
+
+This way, even before you proceed to code review, your CI can tell you if there are any accessibility violations.
+
+## Conclusion
+
+To improve the accessibility of your Angular app:
+
+1. Enable the experimental accessibility rules in codelyzer.
+1. Perform accessibility linting over your entire project using the Angular CLI.
+1. To identify common accessibility issues in your Angular app.
```

Comment:
Second half of the sentence seems to have been lost.

Reply:
I added more details and introduced a fourth bullet point. Let me know if you have any comments.

Reply:
LGTM!